PermissionsService/Classes/Extensions.swift

```diff
@@ -0,0 +1,97 @@
+//
+//  Extensions.swift
+//
+//  Created by Hellen Soloviy on 7/24/17.
+//
+//
+
+import Foundation
+
+extension Int {
+ 
+    func permissionStatus() -> PermissionStatus {
+        
+            switch self {
+                case 0: return .notDetermined
+                case 1: return .restricted
+                case 2: return .denied
+                case 3: return .authorized
+            default: return .unknown
+            
+        }
+        
+    }
+    
+}
+
+extension Optional {
```

Comment:
Remove this extension at all and leave a system app crash which is more informative for user.